Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs

```diff
@@ -191,7 +196,10 @@ private void OnPurchaseMessage(EntityUid uid, ShipyardConsoleComponent component
         var channel = component.ShipyardChannel;
 
         if (ShipyardConsoleUiKey.Security != (ShipyardConsoleUiKey) args.UiKey)
-            _idSystem.TryChangeJobTitle(targetId, $"Captain", idCard, player);
+            if (ShipyardConsoleUiKey.CustomSecurity != (ShipyardConsoleUiKey) args.UiKey)
+                _idSystem.TryChangeJobTitle(targetId, $"Captain", idCard, player);
+            else
+                channel = component.SecurityShipyardChannel;
```

Comment:
this seems off, chould have been an if else or removed from hardcoding

Reply:
Will improve the writing here

Reply:
Pushing the fix now after having tested it.